Dockerfile

```diff
@@ -1,22 +1,25 @@
-FROM docker.osdc.io/ncigdc/python3.8-builder as builder
+ARG REGISTRY=docker.osdc.io
+ARG BASE_CONTAINER_VERSION=2.0.1
+
+FROM ${REGISTRY}/ncigdc/python3.8-builder:${BASE_CONTAINER_VERSION} as builder
 
 COPY ./ /opt
 
 WORKDIR /opt
 
-RUN pip install tox && tox -p
+RUN pip install tox && tox -e build
 
-FROM docker.osdc.io/ncigdc/python3.8
+FROM ${REGISTRY}/ncigdc/python3.8:${BASE_CONTAINER_VERSION}
 
-COPY --from=builder /opt/dist/*.tar.gz /opt
-COPY requirements.txt /opt
+COPY --from=builder /opt/dist/*.whl /opt/
```

Comment:
Strangely, installing from the tar archive was throwing errors related to not finding a suitable `bioinf-maflib` install. Switching to installing wheels fixed this error.